astroquery/utils/tap/core.py

```diff
@@ -27,7 +27,7 @@
 
 __all__ = ['Tap', 'TapPlus']
 
-VERSION = "1.0"
+VERSION = "1.0.1"

```

Comment:
You could also use ``astroquery.__version__`` here if you like, but maybe that' too noisy for the purposes here.